src/sentry/static/sentry/app/views/settings/incidentRules/triggers/chart/thresholdsChart.tsx

```diff
@@ -211,7 +216,7 @@ export default class ThresholdsChart extends React.PureComponent<Props, State> {
     ];
   };
 
-  getChartPixelForThreshold = (threshold: number | '') =>
+  getChartPixelForThreshold = (threshold: number | '' | null) =>
```

Comment:
```suggestion
getChartPixelForThreshold = <V extends number | '' | null>(threshold: V) => {
```

Reply:
Is there a functional difference between these two, or is this more of a style preference? I think the former reads a bit easier because, while reading left to right, it's easier to tell the type of `threshold` without having to backtrack.

Reply:
it's more a style preference. According to the team, what you wrote is easier to read, so let's leave like it is.